packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -6014,13 +6032,20 @@ if(devices.real().deviceType === 'desktop') {
                 helper.checkAttributes(helper.$widget, { }, 'widget');
                 helper.checkAttributes(helper.widget._input(), inputAttributes, 'input');
 
+                helper.widget.option('searchMode', 'startswith');
+                inputAttributes['aria-autocomplete'] = getAriaAutocomplete(searchEnabled, 'startswith');
+
+                helper.checkAttributes(helper.widget._input(), inputAttributes, 'input');
+
                 delete inputAttributes.readonly;
 
                 if(searchEnabled) {
                     inputAttributes.readonly = '';
                 }
 
                 helper.widget.option('searchEnabled', !searchEnabled);
+                inputAttributes['aria-autocomplete'] = getAriaAutocomplete(!searchEnabled, 'startswith');
+
```

Comment:
It's really difficult for me to get from this tests if we check all scenarios, so I gonna trust u about this.
I believe we should split this enormous test to several atomic in the future.

Reply:
[b670242fd1ef0f0f856680661f3802602b72d41c]